src/sockperf.cpp

```diff
@@ -3396,8 +3396,8 @@ packet pace limit = %d",
             s_user_params.fd_handler_type, s_user_params.mthread_server,
             s_user_params.sock_buff_size, s_user_params.threads_num, s_user_params.threads_affinity,
             s_user_params.is_blocked, s_user_params.is_nonblocked_send, s_user_params.do_warmup,
-            s_user_params.pre_warmup_wait, s_user_params.is_vmarxfiltercb,
-            s_user_params.is_vmazcopyread, s_user_params.mc_loop_disable, s_user_params.mc_ttl,
+            s_user_params.pre_warmup_wait, s_user_params.is_xliorxfiltercb,
+            s_user_params.is_xliozcopyread, s_user_params.mc_loop_disable, s_user_params.mc_ttl,
             s_user_params.uc_reuseaddr, s_user_params.tcp_nodelay,
             s_user_params.client_work_with_srv_num, s_user_params.b_server_reply_via_uc,
             s_user_params.b_server_dont_reply, s_user_params.b_server_detect_gaps,
```

Comment:
See lines 3422, 3418
Also didn't find VMA_LIBRARY_MAJOR definition, in libxlio there something like PRJ_LIBRARY_MAJOR.